packages/kit/src/views/AccountManagerStacks/components/WalletRename/WalletRenameButton.tsx

```
@@ -35,93 +35,98 @@ export function WalletRenameButton({
+        flexShrink={1}
+        alignItems="center"
+        borderRadius="$2"
+        {...(canRename && {
+          role: 'button',
+          onPress: async () => {
+            if (!canRename) {
+              return;
+            }
+            if (
+              wallet &&
+              wallet?.id &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)